ckan/logic/action/update.py

```diff
@@ -1314,8 +1312,8 @@ def config_option_update(
 
     unsupported_options = set(provided_options) - set(available_options)
     if unsupported_options:
-        msg = 'Configuration option(s) \'{0}\' can not be updated'.format(
-              ' '.join(list(unsupported_options)))
+        opts = ' '.join(list(unsupported_options))
```

Comment:
simplify while we're here
```suggestion
        opts = ' '.join(unsupported_options)
```